### DISCUSSION TITLE
TextBox IME Mode

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#11553] [Q&A]
Is there a way to set the default IME mode for a TextBox in Avalonia like in WPF?
WPF example:
`<TextBox InputMethod.PreferredImeState="On" InputMethod.PreferredImeConversionMode="Alphanumeric"/>`

**Comment by maxkatz6** [comment]
TextInputOptions.ContentType on 11.0 previews

**Reply by ghost** [reply]
Thx for the reply!
Is there any way without upgrading to the preview version? Still using 0.10 in our project.

**Reply by maxkatz6** [reply]
No

**Reply by ghost** [reply]
Ok, thank you
